src/ucp/core/ucp_mm.c

```diff
@@ -505,8 +508,16 @@ ucs_status_t ucp_memh_register(ucp_context_h context, ucp_mem_h memh,
                                      UCT_MD_MEM_REG_FIELD_DMABUF_OFFSET;
         }
 
-        status = uct_md_mem_reg_v2(context->tl_mds[md_index].md, address,
-                                   length, &reg_params, &memh->uct[md_index]);
+        reg_address = address;
+        reg_length  = length;
```

Comment:
why need to copy to reg_address/reg_length?
IMO better to print log messages with the actual (aligned) address we are passing down to UCT

Reply:
need modify (align) addresses according to specific md before passing down or do I miss something?

Reply:
ok, probably don't want to change parameters.. so it means using reg_address/reg_length in the log/error messages instead of address/length